pkg/cmd/edit/edit_buildpack.go

```diff
@@ -180,3 +193,13 @@ func (o *EditBuildPackOptions) Run() error {
 	}
 	return o.ModifyDevEnvironment(callback)
 }
+
+func (o *EditBuildPackOptions) isBoot(teamSettings *v1.TeamSettings) (*config.RequirementsConfig, error) {
```

Comment:
Missed this Hardy, not sure why we'd like to return an explicit boolean. Is it so the conditional can be simplified to;

`if isBoot` instead of `if isBoot != nil`

I like that a lot, let me update this PR

Reply:
You already did it 😅, thank you!